proto/campaign/campaign.proto

```diff
@@ -22,11 +22,10 @@ message Campaign {
     (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.Coin",
     (gogoproto.castrepeated) = "Shares"
   ];
-  bool dynamicShares = 8;
-  repeated cosmos.base.v1beta1.Coin totalShares = 9 [
+  repeated cosmos.base.v1beta1.Coin totalShares = 8 [
```

Comment:
We remove `totalShares` and related logic as well

`totalShares` was used to specify the new total when `MsgUpdateTotalShares` was used

Reply:
Ok - I can pick this work up once #656 gets merged so I can use some of those additions.